docs/troubleshoot-collection.md

```diff
@@ -340,6 +341,31 @@ You [check Prometheus logs](#prometheus-logs) to verify there are no errors duri
 
 You can also check `prometheus_remote_storage_.*` metrics to look for success/failure attempts.
 
+### Missing metrics for ArgoCD installation
+
+There is known issue with Argo CD and metrics collection. If you override `spec.source.helm.releaseName`, then Kube State and Node metrics are not collected due to the following:
+
+Service Monitor is looking for service labeled with `app.kubernetes.io/instance: <spec.source.helm.releaseName>`, but the label is actually `app.kubernetes.io/instance: <metadata.name>`.
+
+In order to fix it, you need to ensure that the labels are matching and you can do it by adding the following to `user-values.yaml`:
+
+```
+kube-prometheus-stack:
+  kube-state-metrics:
+    prometheus:
+      monitor:
+        selectorOverride:
+          app.kubernetes.io/instance: <metadata.name>
+          app.kubernetes.io/name: kube-state-metrics
+  prometheus-node-exporter:
+    prometheus:
+      monitor:
+        selectorOverride:
+          app.kubernetes.io/name: prometheus-node-exporter
```

Comment:
We cannot use more than label, because our prometheus-node-expoerter is in version 4.3.1, and bug has been fixed in 4.4.2: https://github.com/prometheus-community/helm-charts/pull/2618

which is used from kube-prometheus-stack in version 42.1.1 (version is 4.4.x) or definitely in 45.3.0

We are using kube-prometheus-stack in version `40.5.0`

Reply:
Converted to #3494